functions/Element/getElementPosition.yaml

```diff
@@ -0,0 +1,30 @@
+shared:
+  name: 'getElementPosition'
+  pair: 'setElementPosition'
+  description: |
+    This function allows you to retrieve the position coordinates of an element. This can be any real world element, including:
+    - Players
+    - Vehicles
+    - Objects
+    - Pickups
+    - Markers
+    - Collision shapes
+    - Blips
+    - Radar areas
+    - Syntax
+  returns:
+    description: |
+      Returns three floats indicating the position of the element, x, y and z respectively.
+    values:
+      - type: 'float'
+        name: 'x'
+      - type: 'float'
+        name: 'y'
+      - type: 'float'
+        name: 'z'
+  examples:
+    - path: 'examples/getElementPosition.lua'
+      description: |
+        This example shows how to obtain the position of a player.
```

Comment:
If you fix the source code example, then copy the original description too, please.

Reply:
The examples were quite bad, but alright I'll revert to the original for now.

Reply:
I know, but we should first copy things as they are, before making any drastic changes.